Documentation/Books/Manual/Administration/Configuration/README.md

```diff
@@ -0,0 +1,280 @@
+translated to an option in a configuration file:
+
+```js
+[section]
+param = value
+```
+
+{% hint 'tip' %}
+Whitespace around `=` is ignored in configuration files.
+This includes whitespace around equality signs in the parameter value:
+
+```conf
+log.level = startup = trace
+```
+
+It is the same as without whitespace:
+
+```conf
+log.level=startup=trace
+```
+{% endhint %}
+
+Comments can be placed in the configuration file by placing one or more
+hash symbols `#` at the beginning of a line.
+
+Only command line options with a value should be set within the configuration
+file. Command line options which act as flags should only be entered on the
+command line when starting the server.
+
+## Using Configuration Files
+
+For each binary (except `arangodb`, which is the _Starter_) there is a
+corresponding `.conf` file that an ArangoDB package ships with.
+`arangosh.conf` contains the default ArangoShell configuration for instance.
+The configuration files can be adjusted or new ones be created.
+
+To load a particular configuration file, there is a `--configuration` option
+available to let you specify a path to a `.conf` file. If you want to
+completely ignore a configuration file (likely the default one) without
+necessarily deleting the file, then add the command line option
+
+```
+-c none
+```
+
+or
+
+```
+--configuration none
+```
+
+The value *none* is case-insensitive.
+
+<!-- TODO: Specific to arangod, move to programs detail page?
+     Linux only? Also macOS? Windows not addressed so far.
+
+If this command is not passed to the server, then by default, the server will
+attempt to first locate a file named *~/.arango/arangod.conf* in the user's home
+directory.
+
+If no such file is found, the server will proceed to look for a file
+*arangod.conf* in the system configuration directory. The system configuration
+directory is platform-specific, and may be changed when compiling ArangoDB
+yourself. It may default to */etc/arangodb* or */usr/local/etc/arangodb*. This
+file is installed when using a package manager like rpm or dpkg. If you modify
+this file and later upgrade to a new version of ArangoDB, then the package
+manager normally warns you about the conflict. In order to avoid these warning
+for small adjustments, you can put local overrides into a file
+*arangod.conf.local*.
+
+-->
+
+## Environment variables as parameters
+
+If you want to use an environment variable in a value of a startup option,
+write the name of the variable wrapped in at signs `@`. It acts as a
+placeholder. It can be combined with fixed strings for instance.
+
+Command line example:
+
+```
+arangod --temp.path @TEMP@/arango_tmp
+```
+
+In a configuration file:
+
+```
+[temp]
+path = @TEMP@/arango_tmp
+```
+
+On a Windows system, above setting would typically make the ArangoDB Server
+create its folder for temporary files in `%USERPROFILE%\AppData\Local\Temp`,
+i.e. `C:\Users\xxx\AppData\Local\Temp\arango_tmp`.
+
```

Comment:
Tested successfully on Windows. Does it work the same on Linux/macOS?

The docs previously stated that you would have to write `*Variable*` (did someone accidentally remove the at signs at some point?!)

What special placeholders are there? `@ROOTDIR@` seems to be one and is certainly not an env var.

Reply:
Works on Linux too:
```
export foo=startup
arangod --log.level @foo@=trace
```